### DISCUSSION TITLE
How to prevent this in Windows Edge Webview?

### DISCUSSION THREAD

**Discussion by vednig** [tauri-apps/wry#1359] [Q&A] [answered]
https://github.com/user-attachments/assets/823ca6c0-a01e-4b29-9592-62d81718488b

I understand this is default behavior of Edge WebView, do you think this is possible?

**Comment by FabianLars** [comment, marked as answer]
It's not the same question, but the answer should apply the same: https://github.com/tauri-apps/tauri/discussions/5958

For webview2 specifically you can also set the `--disable-features=ElasticOverscroll` via https://docs.rs/wry/latest/x86_64-pc-windows-msvc/wry/struct.WebViewBuilder.html#method.with_additional_browser_args - can't remember if macos/linux even have the same issue so maybe this native-ish solution is enough.

**Reply by vednig** [reply]
Yeah, it's not reproducible on macOS and Linux

**Reply by vednig** [reply]
Code snippet in the issue doesn't seem to be working, is there any way how can I set the flag in tauri v1 app

**Reply by FabianLars** [reply]
https://docs.rs/tauri/latest/tauri/window/struct.WindowBuilder.html#method.additional_browser_args / https://tauri.app/v1/api/config/#windowconfig.additionalbrowserargs

**Reply by vednig** [reply]
Thank You, Very Much

**Reply by vednig** [reply]
Also, not related but there isn't any documentation for https://github.com/FabianLars/tauri-plugin-oauth is it expected soon?

**Reply by FabianLars** [reply]
Just spoke with someone on Discord today who's interested in contributing js bindings and docs, yes (btw, there are already "docs" in the code if that helps until then).
